Azure/sonic-buildimage
# 20210616 as9716 32d 202012 br sfputil py

#### Why I did it
sfputil eeprom need to support QSFP-DD
#### How I did it
Include module(inf8628.py)
#### How to verify it
Use the command "sfputil show eeprom" to verify get eeprom info is correct.
Ex:
Ethernet96: SFP EEPROM detected
        Identifier: QSFP-DD Double Density 8X Pluggable Transceiver
        Interrupt Asserted: On
        Module State: Unknown State 0
        Revision Compliance: 3.0
        Vendor Name: EdgecoreNetworks
        Vendor Part Number: ET7502-DAC-2.5M
        Vendor Revision: A0
        Vendor Serial Number: J22033303008

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

This PR only want to merge on 202012 branch.

#### A picture of a cute animal (not mandatory but encouraged)
